tools/main.c

```diff
@@ -69,8 +69,15 @@ main (int   argc,
         showVersion (argv[0]);
         exit (0);
     }
+
     if (opts.verbose)
         log_set_level(log_level_verbose);
```

Comment:
Does the dis-coupling of verbose and quiet do what you want when quiet is enabled? I'm ok with the behavior, as it matches grep:
```
echo foo | grep ",[" -q
grep: Invalid regular expression
```